library/src/scala/CanEqual.scala

```diff
@@ -29,6 +29,8 @@ object CanEqual {
   // The next three definitions can go into the companion objects of classes
```

Comment:
```suggestion
  // The next 4 definitions can go into the companion objects of classes
```

Reply:
Oh, that's good. I'll update it soon. Thanks!